src/main/java/bartworks/system/material/werkstoff_loaders/recipe/AspectLoader.java

```diff
@@ -37,18 +37,18 @@ public void run(Werkstoff werkstoff) {
 
         for (OrePrefixes enabledOrePrefixes : WerkstoffLoader.ENABLED_ORE_PREFIXES) {
             if (werkstoff.hasItemType(enabledOrePrefixes)) {
-                if (enabledOrePrefixes.mMaterialAmount >= 3628800L || enabledOrePrefixes == OrePrefixes.ore) {
+                if (enabledOrePrefixes.getMaterialAmount() >= 3628800L || enabledOrePrefixes == OrePrefixes.ore) {
                     DebugLog.log(
-                        "OrePrefix: " + enabledOrePrefixes.name()
+                        "OrePrefix: " + enabledOrePrefixes.getName()
                             + " mMaterialAmount: "
```

Comment:
Perhaps this should be changed to match?
```suggestion
                            + " Material Amount: "
```